src/psyclone/psyir/nodes/operation.py

```diff
@@ -138,6 +148,8 @@ class UnaryOperation(Operation):
         'REAL', 'INT', 'NINT'
         ])
 
+    _non_elemental_names = ['SUM']
```

Comment:
I think this would be nicer if it was `_non_elemental_ops = [Operator.SUM]` rather than using strings. Is that possible?

Reply:
Yes, and I agree. Done.